### ISSUE TITLE
Nightvision doesn't function correctly at midnight

### ISSUE THREAD

**Issue by Hypno123** [deathlyrage/theisle-bugs#386]
Patch: 5225
Server: US Official 2
Issue: Nightvision doesn't function correctly at midnight.
There is almost no difference in shadows when turning nightvision on and off, as shown in the video.

- posting this here for reference on the commonly reported bugs document.

Video taken on Tigerlily, however it's apparent on all servers: 
https://www.youtube.com/watch?v=yEvIcH_s_aE

**Comment by Souretsu** [comment]
I believe this is related to shadows, as it is not as apparent in some areas of the map. Will update as I look into it

**Comment by Hypno123** [comment]
Yeah, it's a known issue - I was posting simply for a reference on a document I was working on.
